app/pandas.ipynb

```diff
@@ -271,54 +601,197 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "outputs": [],
    "source": [
     "# Data\n",
     "data1 = {'Name': ['John', 'Emily', 'Ryan'],\n",
     "         'Age': [25, 28, 22]}\n",
     "data2 = {'Name': ['Emily', 'Ryan', 'Mike'],\n",
     "         'City': ['Los Angeles', 'Chicago', 'Houston']}\n",
     "\n",
-    "# create DataFrame objects"
+    "dat1 = pd.DataFrame(\n",
+    "    data1\n",
+    ")\n",
+    "dat2 = pd.DataFrame(\n",
+    "    data2\n",
+    ")\n"
```

Comment:
The creation of `dat1` and `dat2` DataFrames is redundant since they are identical to `data1` and `data2`. You can simplify your code by directly creating DataFrames from `data1` and `data2` without redefining them.